app/src/main/java/org/alexcode/snake/PlayerPreferences.java

```diff
@@ -54,14 +56,36 @@ public static int getHiScore() {
         return sharedPreferences.getInt("HI_SCORE", 0);
     }
 
-    public static String getVolume() {
```

Comment:
why did you remove the volume though?

Reply:
I split the sounds part in two: game music and fx sounds.